src/utils/cuentas_processor_utils.py

```diff
@@ -18,6 +18,7 @@ def read_data(file_path: Union[str, Path]) -> pd.DataFrame:
     try:
         return pd.read_csv(file_path, sep=';', encoding='latin_1', dtype=str)
     except Exception:
+        logging.exception("Failed to read csv")
```

Comment:
`logging.exception` are capturated as error level ?

Reply:
It logs the error and the traceback to the terminal, and sentry capture it, but because the exception is not raised it doesn't stop the program execution. 
 >exception(msg, *args, **kwargs)
    Logs a message with level ERROR on this logger.
https://docs.python.org/3/library/logging.html#logging.Logger.exception
<img width="1553" alt="Screenshot 2023-06-13 at 19 02 05" src="https://github.com/alquimistas-org/subida_cartera/assets/30585029/d46251ac-fb36-4649-9968-66a5c43b4697">